backend/app2/llm/adapters/openai_adapter.py

```
@@ -0,0 +1,181 @@
+from typing import List, Any, AsyncGenerator, Dict, Optional, Sequence
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)